swap/src/cli/command.rs

```diff
@@ -111,15 +108,13 @@ pub enum Command {
 pub struct AliceConnectParams {
     #[structopt(
         long = "seller-peer-id",
-        default_value = DEFAULT_ALICE_PEER_ID,
-        help = "The peer id of a specific swap partner can be optionally provided"
+        help = "The sellers peer identity eg. 12D3KooWCdMKjesXMJz1SiZ7HgotrxuqhQJbP5sgBm2BwP1cqThi"
     )]
     pub peer_id: PeerId,
 
     #[structopt(
         long = "seller-addr",
-        default_value = DEFAULT_ALICE_MULTIADDR,
-        help = "The multiaddr of a specific swap partner can be optionally provided"
+        help = "The sellers multiaddress eg. /dns4/xmr-btc-asb.coblox.tech/tcp/9876"
```

Comment:
I think it would be better to have these values in the changelog instead. The CLI should be completely unbiased IMO.